README.de.md

```diff
@@ -0,0 +1,97 @@
+| Du sind an eine bestimmte Marke gebunden                                                                | Dlool ist kostenlos und quelloffen. *(Mit Dlool können Du Ihre Daten in ical oder todo.txt exportieren. Außerdem gibt es [eine Möglichkeit](https://github.com/Dlurak/dlool-scriptable/blob/main/reminders/README.md), Dlool mit Apple Reminders zu synchroniDuren.)* **(noch nicht implementiert)** |
+| Hausaufgaben, Tests und Notizen sind nicht getrennt                                                      | Dlool hat ein anderes System für Hausaufgaben, Tests und Notizen. Alles ist getrennt organiDurt, aber dennoch leicht an einem Ort zu finden.                                                                                                                                                          |
+
+---
+
+## Feature
+
+Dlool bietet eine Vielzahl von Funktionen, um Ihre Hausaufgabenverwaltung zu verbessern.
+
+- **Kollaborativ**: Alle Einträge aus einer Klasse sind für alle verfügbar.
+- **Strukturiert**: Dlool hat ein strukturiertes Eingabesystem. Es ist einfach zu bedienen, aber auch sehr leistungsfähig.
+- *(***Exportierbar**: Du können Ihre Daten nach ical oder todo.txt exportieren,
+  mit [diesem] (https://github.com/Dlurak/dlool-scriptable/blob/main/reminders/README.md) Skript können Du Dlool auch mit Apple Reminders synchroniDuren.)* **(noch nicht implementiert)**
+- **Plattformübergreifend**: Zugriff auf Dlool von allen Geräten mit einem Browser.
+- **DunkelmoDus**: Dlool passt sich automatisch an Ihre Systemeinstellungen an. Dlool respektiert auch reDuzierte Bewegungseinstellungen.
+- **REST-API**: Es gibt eine offene REST-API, um eigene Skripte zu schreiben.
+- **Kein Konto erforderlich**: Auch ohne Konto können Du Dlool nutzen.
+- **Langfristige Nutzbarkeit**: Halten Du Ihre Daten über Jahre hinweg organiDurt.
+- **Moderationssystem**: Erhalten Du eine positive, einladende und sichere Umgebung aufrecht.
+- **Notizen**: Schreiben Du kurze Notizen für sich oder Ihre Klasse.
+- **Kalender**: Behalten Du den Überblick über Tests und Fristen mit dem wunderschönen Kalender von Dlool.
+- **Stundenplan**: Mit der Stundenplanunterstützung weißt Du ganz einfach, welche Fächer wann sind und kannst die Hausaufgaben richtig aufschreiben.
+- **Feiertage**: Schauen Du nach, wann in Ihrer Gegend Feiertage sind, die Daten werden von [der OpenHolidayAPI](https://www.openholidaysapi.org/en/) bereitgestellt
+  
+- Viele kleine Dinge, die Ihnen das Leben leichter machen.
+
+---
+
+## Technologien
+
+### Frontend
+
+Dlool ist eine progressive WebanwenDung und wird von den folgenden Technologien unterstützt:
```

Comment:
- "WebanwenDung" with a capital D
- wird von den folgenden Technologien unterstützt sounds more like sveltekit, tailwind... support us but actually Dlool is built on top of them